apps/fluent-tester/README.md

```diff
@@ -5,5 +5,5 @@
 - [iOS](./../ios/README.md)
 - [Android](./../android/README.md)
 - [macOS](./../macos/README.md)
-- [Windows](./../windows/README.md)
-- [Win32](./../win32/README.md)
\ No newline at end of file
+- [Windows](./../windows/readme.md)
+- [Win32](./../win32/README.md)
```

Comment:
Could we instead just capitalize the name to README? Since every other platform seems to do that